### PR TITLE
docs: redoing for what was pr 4672

### PR DESCRIPTION
**Pull request by msdolbey**
C

https://d2iq.atlassian.net/browse/D2IQ-94862

### Preview

You can preview the docs build using the following URL, adding the PR # where specified:

http://docs-d2iq-com-pr-4676.s3-website-us-west-2.amazonaws.com/ 

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

File: pages/dkp/kommander/2.2/release-notes/2.2.0/index.md
Comment (msdolbey) [inline pages/dkp/kommander/2.2/release-notes/2.2.0/index.md:358]: Should it get called out sooner that this is only on Flatcar?

```suggestion
### Calico not updated during DKP upgrade on Flatcar
```

File: pages/dkp/kommander/2.2/release-notes/2.2.0/index.md
Comment (msdolbey) [inline pages/dkp/kommander/2.2/release-notes/2.2.0/index.md:360]: From @cneth 

```suggestion
When upgrading a DKP cluster running on Flatcar OS, you may find that after the upgrade the Calico services were not updated. This occurs because the upgrade procedure is not correctly updating the Flatcar specific CNI ClusterResourceSet(CRS). This issue only impacts the Calico CRS.
```

File: pages/dkp/kommander/2.2/release-notes/2.2.0/index.md
Comment (msdolbey) [inline pages/dkp/kommander/2.2/release-notes/2.2.0/index.md:398]: @dkoshkin @cneth - I just committed the CLUSTER_NAME business and the clusterresourceset here

File: pages/dkp/kommander/2.2/release-notes/2.2.1/index.md
Comment (msdolbey) [inline pages/dkp/kommander/2.2/release-notes/2.2.1/index.md:117]:
```suggestion
### Calico not updated during DKP upgrade on Flatcar
```

File: pages/dkp/kommander/2.2/release-notes/2.2.2/index.md
Comment (msdolbey) [inline pages/dkp/kommander/2.2/release-notes/2.2.2/index.md:66]:
```suggestion
### Calico not updated during DKP upgrade on Flatcar
```

File: pages/dkp/kommander/2.2/release-notes/2.2.3/index.md
Comment (msdolbey) [inline pages/dkp/kommander/2.2/release-notes/2.2.3/index.md:99]:
```suggestion
### Calico not updated during DKP upgrade on Flatcar
```

File: pages/dkp/kommander/2.2/release-notes/2.2.0/index.md
Comment (cmastr) [inline pages/dkp/kommander/2.2/release-notes/2.2.0/index.md:356]:
```suggestion
When upgrading DKP from v2.1.x to v2.2.x, the upgrade can fail due to insufficient space on the MinIO Disk. To avoid this issue, D2iQ recommends that you disable the `fluent-bit` Platform Application before upgrading.
```

File: pages/dkp/kommander/2.2/release-notes/2.2.0/index.md
Comment (cmastr) [inline pages/dkp/kommander/2.2/release-notes/2.2.0/index.md:360]:
```suggestion
When upgrading a DKP cluster, after what seems to be a successful upgrade, the Calico service may not update as expected and, therefore, is still using an old image. The wrong CNI ClusterResourceSet is being generated and not accounting for Flatcar. This issue only impacts Calico, no other applications.
```

File: pages/dkp/kommander/2.2/release-notes/2.2.0/index.md
Comment (cmastr) [inline pages/dkp/kommander/2.2/release-notes/2.2.0/index.md:398]:
```suggestion
1.  Run these commands: 

`kubectl edit clusterresourceset calico-cni-installation-$CLUSTER_NAME` and update `spec.clusterSelector.matchLabels.konvoy.d2iq.io/osHint` to `konvoy.d2iq.io/osHint: flatcar`.
```

File: pages/dkp/kommander/2.2/release-notes/2.2.1/index.md
Comment (cmastr) [inline pages/dkp/kommander/2.2/release-notes/2.2.1/index.md:36]:
```suggestion
An issue with the ClusterResourceSet controller in 2.2.0 causes an unbounded number of service account tokens to be created for each ClusterResourceSet. This problem has been corrected. A remediation is also available to identify and remove the excess secrets; see this [knowledge base](https://support.d2iq.com/hc/en-us/articles/6019137621908-Customer-Advisory-D2iQ-2022-0002-Unbounded-Number-of-Service-Account-Token-Secrets-Created) article for more information.
```

File: pages/dkp/kommander/2.2/release-notes/2.2.1/index.md
Comment (cmastr) [inline pages/dkp/kommander/2.2/release-notes/2.2.1/index.md:111]:
```suggestion
If any of the these fields are present, the upgrade can fail. If you encounter this situation, contact D2iQ Support for advice on how to remediate the issue before attempting to continue the upgrade.
```

File: pages/dkp/kommander/2.2/release-notes/2.2.1/index.md
Comment (cmastr) [inline pages/dkp/kommander/2.2/release-notes/2.2.1/index.md:119]: See notes above and change this to be in line with other item.

File: pages/dkp/kommander/2.2/release-notes/2.2.0/index.md
Comment (cneth) [reply pages/dkp/kommander/2.2/release-notes/2.2.0/index.md:360]: @msdolbey @cmastr - we need to pick one of these two and then make sure they get carried over to the 2.3/2.4/2.5 pages:

https://d2iq.atlassian.net/wiki/spaces/DENT/pages/150831110/DKP+2.5.0+Known+Issues+and+Limitations

Comment (cmastr) [reply pages/dkp/kommander/2.2/release-notes/2.2.0/index.md:360]: Gonna commit Matt's then make new changes. We will need a JIRA to track 2.5, or we'll need to copy paste it now before we merge this PR.

**Review by cmastr — APPROVED** [review]

**Comment by cmastr** [comment]
2.4 done here - https://d2iq.atlassian.net/wiki/spaces/DENT/pages/150831110/DKP+2.5.0+Known+Issues+and+Limitations#Calico-Not-Updated-During-DKP-Upgrade-on-Flatcar

**Review by Colin1841 — APPROVED** [review]

**Comment by Colin1841** [comment]
We might end up deleting this and just fixing the issue, so leaving this as DO NOT MERGE for now and we'll figure this out in January.

**Review by Colin1841 — APPROVED** [review]

**Pull request merged**
